test/ext_hash_test.rb

```diff
@@ -19,16 +19,6 @@ def test_to_params_array
     assert_equal "foo[0]=bar&foo[1]=boo", { :foo => [ :bar, :boo ] }.to_params
   end
 
-  def test_to_params_crazy
```

Comment:
This test was actually unnecessary. I was testing to make sure bad data was properly escaped, however, it actually shouldn't be and an error should be thrown. When using `Addressable::URI.escape` it will be.